spec/integration/schema_rotation_spec.rb

```diff
@@ -18,6 +18,19 @@
     end
   end
 
+  it 'reads and writes the embeddings table from the public schema' do
```

Comment:
this definitely isn't a full test of all paths -- doing that would add a lot of overhead though, so i'm not sure if there's a good way to do this with the current integration-test heavy approach. this basically just checks that all of the embedded checks that happen when we process a message work without creating an embedding in this schema.